.prettierrc

```diff
@@ -0,0 +1,6 @@
+{ 
+  "semi": false,
+  "tabWidth": 2,
```

Comment:
Do we want tabs or spaces? Just to keep it consistent with styling on the backend

Reply:
Lets use tabs

Reply:
will change that, thx

Reply:
I have change it to 4, please review again